common/src/main/java/com/wynntils/features/user/players/PreventTradesDuelsFeature.java

```diff
@@ -14,25 +16,39 @@
 import net.minecraft.world.entity.Entity;
 import net.minecraft.world.entity.player.Player;
 import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.eventbus.api.Event;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
 @FeatureInfo(category = FeatureCategory.PLAYERS)
 public class PreventTradesDuelsFeature extends UserFeature {
+    private static final int FIGHTING_CUTOFF = 30; // 30 seconds
```

Comment:
Why not a config?

Reply:
I did not see the need to change this on a per-user basis. If the value is bad, I think we should adjust it for all users. I did some simple testing with me and my alt account and I thought this seemed like a good tradeoff between not having to wait too long but still keep the prevention while actively engaged. Too many config just makes everything confusing.

Are you happy with that reply or do you want me to turn it into a config?

Reply:
Okay. 30 seconds sounds reasonable. Let's leave it.